GPT/beta-commands/continue.talon

```diff
@@ -4,19 +4,37 @@ tag: user.gpt_beta
 
 ## Commands to interact with https://continue.dev/ extension
 
-tin you new: user.vscode("continue.newSession")
-tin you file select: user.vscode("continue.selectFilesAsContext")
-tin you history: user.vscode("continue.viewHistory")
-tin you (accept | yes): user.vscode("continue.acceptDiff")
-tin you reject: user.vscode("continue.rejectDiff")
-tin you toggle fullscreen: user.vscode("continue.toggleFullScreen")
-tin you cancel: key("escape")
-tin you debug terminal: user.vscode("continue.debugTerminal")
+tin you new:                user.vscode("continue.newSession")
+tin you file select:        user.vscode("continue.selectFilesAsContext")
+tin you history:            user.vscode("continue.viewHistory")
+tin you (accept | yes):     user.vscode("continue.acceptDiff")
+tin you reject:             user.vscode("continue.rejectDiff")
+tin you toggle fullscreen:  user.vscode("continue.toggleFullScreen")
+tin you cancel:             key("escape")
+tin you debug terminal:     user.vscode("continue.debugTerminal")
+tin you share:              user.vscode("continue.shareSession")
+tin you select files:       user.vscode("continue.selectFilesAsContext")
 tin you add <user.cursorless_target>:
-    user.cursorless_command("setSelection", cursorless_target)
-    user.vscode("continue.focusContinueInput")
+    user.cursorless_ide_command("continue.focusContinueInput", cursorless_target)
 tin you edit <user.cursorless_target>:
-    user.cursorless_command("setSelection", cursorless_target)
-    user.vscode("continue.quickEdit")
+    user.cursorless_ide_command("continue.quickEdit", cursorless_target)
 
-bar tin you: user.vscode("continue.continueGUIView.focus")
+tin you dock string <user.cursorless_target>:
+    user.cursorless_ide_command("continue.writeDocstringForCode", cursorless_target)
+
+tin you comment <user.cursorless_target>:
+    user.cursorless_ide_command("continue.writeCommentsForCode", cursorless_target)
+
+tin you comment <user.cursorless_target>:
+    user.cursorless_ide_command("continue.writeCommentsForCode", cursorless_target)
+
+tin you optimize <user.cursorless_target>:
+    user.cursorless_ide_command("continue.optimizeCode", cursorless_target)
+
+tin you fix code <user.cursorless_target>:
+    user.cursorless_ide_command("continue.fixCode", cursorless_target)
+
+tin you fix grammar <user.cursorless_target>:
+    user.cursorless_ide_command("continue.fixGrammar", cursorless_target)
```

Comment:
Is `continue.fixGrammar` your custom action? Was curious just since we have `model fix grammar` obviously. Wasn't sure if coincidence.

Reply:
It's one of the built in commands from continue